edx_name_affirmation/apps.py

```diff
@@ -25,13 +25,9 @@ class EdxNameAffirmationConfig(AppConfig):
                 'relative_path': 'handlers',
                 'receivers': [
                     {
-                        'receiver_func_name': 'idv_attempt_handler',
-                        'signal_path': 'lms.djangoapps.verify_student.signals.signals.idv_update_signal',
-                    },
-                    {
-                        'receiver_func_name': 'idv_delete_handler',
```

Comment:
We don't really need this anymore since once moved over to the new model `SoftwareSecurePhotoVerification`s aren't getting created anymore. Those that are in `pending` are now manually approved so there's no real point in having the ability to delete the record and retry.